src/libraries/Microsoft.PowerFx.Connectors/Tabular/CdpTableResolver.cs

```diff
@@ -83,6 +88,21 @@ public async Task<CdpTableDescriptor> ResolveTableAsync(string tableName, Cancel
                     // Result should be cached
                     sqlRelationships = GetSqlRelationships(text2);
                 }
+                else if (IsOracle())
```

Comment:
>IsOracle [](http://example.com/codeflow?start=25&length=8)

Let's followup. 
This code really ought to be server-side int he CDP implementation; and not client-side here.